internal/cli/root/builder.go

```diff
@@ -96,3 +111,55 @@ func formattedVersion() string {
 		runtime.GOARCH,
 		runtime.Compiler)
 }
+
+func (opts *BuilderOpts) hasNewVersionAvailable() (newVersionAvailable bool, newVersion string, err error) {
+	if version.Version == "" {
+		return false, "", nil
+	}
+
+	svCurrentVersion, err := semver.NewVersion(version.Version)
+	if err != nil {
+		return false, "", err
+	}
+
+	latestVersion, err := opts.store.LatestVersion()
+	if err != nil {
+		return false, "", err
+	}
+
+	svLatestVersion, err := semver.NewVersion(latestVersion)
+	if err != nil {
+		return false, "", err
+	}
+
+	if svCurrentVersion.Compare(svLatestVersion) < 0 {
+		return true, latestVersion, nil
+	}
+
+	return false, "", nil
+}
+
+func (opts *BuilderOpts) printNewVersionAvailable(w io.Writer) error {
+	if config.SkipUpdateCheck() {
+		return nil
+	}
+	newVersionAvailable, latestVersion, err := opts.hasNewVersionAvailable()
+	if err != nil {
+		return err
+	}
+	if newVersionAvailable {
+		newVersionTemplate := `
+A new MongoCLI version is available '%s'!
+To upgrade, check https://docs.mongodb.com/mongocli/stable/install/ (run "mongocli config set skip_update_check true" to disable the alert.)
```

Comment:
from the spec it mentions defining a new page to explain how to upgrade, docs has ways to generate links that don't depend on a page actually existing, get to them to get one of those for this

Reply:
PS: side note, you can know is a homebrew installation, in which case offering the command `hombrew update && hombrew upgrade mongocli` may be more helpful

Reply:
PS2: let's make sure the version checking works with homebrew installations

Reply:
So I'm opening a [ticket](https://jira.mongodb.org/browse/CLOUDP-100237) for this, I think this would make the alert much better but also we would need if the latest version was already bumped in homebrew repo before alerting.

Since this new check might be more involved I'll do it in another PR, if that is OK with you @gssbzn?

Reply:
works for me

Reply:
I created a dochub link which points to https://docs.mongodb.com/mongocli/stable/install/ and added a few wording suggestions:
```suggestion
To upgrade MongoCLI, see https://dochub.mongodb.org/core/mongocli-install . To disable this alert, run "mongocli config set skip_update_check true" .)
```